src/database.py

```diff
@@ -0,0 +1,37 @@
+import os
+import psycopg2
+import logging
+import sys
+
+# global singleton connection
+_connection = None
+
+
+def connect():
+    """ Connect to the PostgreSQL database server"""
+    global _connection
+    if not _connection:
+        try:
+            # connect to the PostgreSQL server
+            logging.info("Connecting to the PostgreSQL database...")
+            _connection = psycopg2.connect(host=os.getenv("HOST"),
+                                           database=os.getenv("DATABASE"),
+                                           user=os.getenv("DB_USER"),
+                                           password=os.getenv("DB_PASSWORD"))
```

Comment:
Prefix all the environment variables with `RC_DINING_BOT_`

Reply:
Oh makes sense

Reply:
Rmb to change all environment variables, not limited to these 4